src/main/charts/bitbucket/values.yaml

```diff
@@ -567,6 +562,17 @@ bitbucket:
       #
       passwordSecretKey: password
 
+  shutdown:
+    # -- The termination grace period for pods during shutdown. This
+    # should be set to the Bitbucket internal grace period (default 30
+    # seconds), plus a small buffer to allow the JVM to fully terminate.
+    terminationGracePeriodSeconds: 35
+
+    # By default, Bitbucket will be stopped by the script supplied by the image.
```

Comment:
Should we expand on this comment and include a reference/url to what this script is?

Reply:
Done